src/taxonomy/import-tags/ImportTagsWizard.jsx

```diff
@@ -25,12 +26,15 @@ import LoadingButton from '../../generic/loading-button';
 import { LoadingSpinner } from '../../generic/Loading';
 import { getFileSizeToClosestByte } from '../../utils';
 import { TaxonomyContext } from '../common/context';
-import { getTaxonomyExportFile } from '../data/api';
-import { useImportTags, useImportPlan } from '../data/apiHooks';
+import { getTaxonomyExportFile, apiUrls } from '../data/api';
+import { useImportTags, useImportPlan, useImportNewTaxonomy } from '../data/apiHooks';
 import messages from './messages';
 
 const linebreak = <> <br /> <br /> </>;
 
+const csvTemplateUrl = <a href={apiUrls.taxonomyTemplate('csv')} download>CSV template</a>;
+const jsonTemplateUrl = <a href={apiUrls.taxonomyTemplate('json')} download>JSON template</a>;
```

Comment:
These strings should be in messages so they can be extracted for translation: "CSV template" , "JSON template"